src/Service/Consignment/ConsignmentService.php

```diff
@@ -541,4 +536,45 @@ public function findManyByReferenceId(array $referenceIds): MyParcelCollection
     {
         return MyParcelCollection::findManyByReferenceId($referenceIds, $this->apiKey);
     }
+
+    /**
+     * @param  string $shippingOptionId
+     *
+     * @return OrderEntity|null
+     */
+    public function getFullOrderByShippingOptionId(string $shippingOptionId): ?OrderEntity
+    {
+        $options = $this->shippingOptionsService->getShippingOptions(
+            $shippingOptionId,
+            new Context(new SystemSource())
```

Comment:
Dit ziet er een beetje raar uit, wat is dit?

Reply:
Dit heb ik simpelweg overgenomen, ‘blijkbaar’ is dit de manier waarop een context gemaakt wordt in Shopware6 wanneer die niet doorgegeven is.

Reply:
ew...